metricbeat/module/windows/_meta/config.yml

```diff
@@ -0,0 +1,13 @@
+- module: windows
+  metricsets: ["perfmon"]
+  enabled: true
+  period: 1s
+  counters:
```

Comment:
For metricset specific configs we (recently) created the convention to have it under a namespace. Means here it would be `perfmon.counters:`